Kickstarter-iOS/Views/Controllers/ProjectActivityViewControllerTests.swift

```diff
@@ -54,7 +54,7 @@ internal final class ProjectActivityViewControllerTests: TestCase {
   }
 
   func testVoiceOverRunning() {
-    withEnvironment(isVoiceOverRunning: { true }) {
+    withEnvironment(isVoiceOverRunning: const(true)) {
```

Comment:
New swiftlint rule requires that functions with multiple function parameters avoids trailing closure syntax. For things like `either(ifLeft: { … }, ifRight: { … })` I like this, so keeping it enabled and switching these to use `const`.